.prettierrc.json

```diff
@@ -1,6 +1,5 @@
 {
   "printWidth": 80,
   "semi": true,
-  "singleQuote": true,
```

Comment:
`azure/pipelines/*.yml` has "$(foo)" sections, that if reverted to `$(foo)`, it might use it literally, instead of expanding it

To avoid single+double quote mismatch, just standardized to the default, which is double